### ISSUE TITLE
Rename produces double name in shortcut refinement syntax

### ISSUE THREAD

**Issue by lucaswerkmeister** [ceylon/ceylon-ide-eclipse#1020]
```ceylon
interface Sup {
    shared formal Anything f;
}
class Sub() satisfies Sup {
    f => 0;
}
```
If you Rename (Alt+Shift+R, at least on my system) `f` to, say, `g`, you get
```ceylon
interface Sup {
    shared formal Anything g;
}
class Sub() satisfies Sup {
    gg => 0;
}
```
`Sub.f` got changed to `gg` rather than `g`.

**Comment by gavinking** [comment]
I have reverted the fix, which was the cause of #1422, and I'm reopening this issue.

**Comment by gavinking** [comment]
OK, now refixed.
